ui/conf_ui.go

```diff
@@ -33,7 +37,7 @@ func NewWrappingConfUI(parent UI, logger ExternalLogger) *ConfUI {
 }
```

Comment:
@cppforlife IDK if we should change the `isTTY` value here as well.
There is a behavior change here. If somebody initializes  `NewWrappingConfUI ` and then call `EnableTTY()` function with param `force` set to false, then we will be overriding the parent. What do you think of this behavior change ?